### PR TITLE
Miscellaneous documentation fixes

### PR DESCRIPTION
**Pull request by Shashi456**
Some basic formatting touchups and I added a line about data inconsistencies to the Structure of project part of the project. I also added a line just saying that pre-train or train.py can be run locally afterwards. One point i had regarding formatting is, shouldn't all heading be H2 instead of H3?

@marco-c

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +56,8 @@ For the unsupervised training, we are using a related problem for which we alrea
 - The **collect.py** script is the crawler that collects screenshots of web pages in different browsers;
 - The **label.py** script is a utility that helps labelling couples of screenshots (are they the same in the two browsers or are there differences?);
 - The **pretrain.py** script trains a neural network on the website screenshots for a slightly different problem (for which we know the solution), so that we can reuse the network weights for the training on the actual problem;
-- The **train.py** script trains the neural network on the website screenshots to detect compat issues.
+- The **train.py** script trains the neural network on the website screenshots to detect compat issues;
+- The **data_inconsistencies.py** script checks the data/ repository and writes the inconsistencies to an inconsistencies.csv file
```
Comment (marco-c) [inline README.md]: `The **data_inconsistencies.py** script checks the generated screenshots and takes note of any data inconsistency (e.g. screenshots that were taken in Firefox but not in Chrome).`

File: README.md [hunk 2]
```diff
@@ -64,6 +66,7 @@ For the unsupervised training, we are using a related problem for which we alrea
 - Install the dependencies in requirements.txt: `pip install -r requirements.txt`.
 - Install the dependencies in test-requirements.txt: `pip install -r test-requirements.txt`.
 - Run the **get_dependencies.py** script.
+- Run the **pre-train.py / train.py** script, for locally running the neural network.
```
Comment (marco-c) [inline README.md]: - Run the **pretrain.py** or *train.py** script to train the neural network.

File: README.md [hunk 3]
```diff
@@ -64,6 +66,7 @@ For the unsupervised training, we are using a related problem for which we alrea
 - Install the dependencies in requirements.txt: `pip install -r requirements.txt`.
 - Install the dependencies in test-requirements.txt: `pip install -r test-requirements.txt`.
 - Run the **get_dependencies.py** script.
+- Run the **pre-train.py / train.py** script to train the neural network.
```
Comment (marco-c) [inline README.md]: The name of the file is `pretrain.py` and not `pre-train.py`.

You can write this:
```
- Run the **pretrain.py** / **train.py** script to train the neural network.
```

File: README.md [hunk 4]
```diff
@@ -55,7 +56,8 @@ For the unsupervised training, we are using a related problem for which we alrea
 - The **collect.py** script is the crawler that collects screenshots of web pages in different browsers;
 - The **label.py** script is a utility that helps labelling couples of screenshots (are they the same in the two browsers or are there differences?);
```
[the remark this answers is not in the document]
Comment (Shashi456) [reply README.md]: @sagarvijaygupta would that be needed to locally run the neural net? or does get dependencies take care of that?

[the remark this answers is not in the document]
Comment (marco-c) [reply README.md]: > Should we add that label.py should be run as python label.py <name>? @marco-c

Yes, we can do it in a separate PR.

> @sagarvijaygupta would that be needed to locally run the neural net? or does get dependencies take care of that?

Labeling and training are two separate steps. Labeling is not required to run **pretrain.py**, but it is for running **train.py**.
`get_dependencies.py` doesn't do anything related to labels. It only downloads tools and screenshots.
Anyway, what @sagarvijaygupta was suggesting is unrelated to this.
